scripts/save-test-data.php

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/env php
+<?php
+
+use LibreNMS\Proc;
+
+global $device;
+
+$install_dir = realpath(__DIR__ . '/..');
+chdir($install_dir);
+
+$options = getopt(
+    'h:dm:o:v:f:',
+    array(
+        'debug',
+        'no-save',
+        'hostname:',
+        'help',
+        'module:',
+        'os:',
+        'variant:',
+        'file:',
+    )
+);
+
+$debug = (isset($options['d']) || isset($options['debug']));
+$vdebug = $debug;
+$snmpsim_ip = '127.1.6.1';
```

Comment:
I've not come across 127.1.6.1 used as a localhost IP before, is it sensible to use it over 127.0.0.1?